content/participants/ttcj@Adhikaribharath/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Bharath 
+type: "participant"
+name: "Bharath Adhikari"
+portfolio: "null"
+github-link: "https://github.com/Adhikaribharath"
+linkedin-link: "https://www.linkedin.com/in/bharath-kumar-adhikari-b53b81125/"
+looking-for: "job"
+description: "Good with backend stack(java,API's,..). I have good work experience on java,webservices."
+---
+
+Hi! I am Bharath working in a startup, I have got 2years of professional work exprerience. Let me walk you through my journey so far.
+
+### introduction
+
+I am a software developer wth 2 years of expertise in java and backend technologies. I have worked in one of the MNC's for close to 1.6 years and moved to startup recently.
+
+
+## why are you taking this challenge?
+
+During my complete experience I haven't got a chance to work on any of the current front end technologies. Seeing so much of expertise in this team for UI, I thought this would be best place to learn frontend which gives me a chance to stand as a FULL Stack Developer.
+
+## something interesting about you?
+
+I am 24 and I enjoy playing Dj.
+
+### help I need?
+
+help me add Frontend technologies to my Tech stack.
+
+### how to contact me?
+
+- [Twitter]()
```

Comment:
Insert the twitter link or simply remove it

Reply:
Hi I have removed the twitter link.